barretenberg/cpp/pil/vm2/ff_gt.pil

```diff
@@ -0,0 +1,216 @@
+include "./range_check.pil";
+
+// This module handles field gt
+// GT also enables us to support LT (by swapping the inputs of GT) and LTE (by negating the result of GT)
+// Lifecycle table:
+// +-----+-----+--------+--------------+--------------+------------+------------+------+------+------------+------------+--------+--------+-------------+-----+--------+---------------+
+// |  a  |  b  | result | a_hi (range) | a_lo (range) | p_sub_a_hi | p_sub_a_lo | b_hi | b_lo | p_sub_b_hi | p_sub_b_lo | res_hi | res_lo | cmp_rng_ctr | sel | sel_gt | sel_shift_rng |
+// +-----+-----+--------+--------------+--------------+------------+------------+------+------+------------+------------+--------+--------+-------------+-----+--------+---------------+
+// | 0   | 0   | 0      | 0            | 0            | 0          | 0          | 0    | 0    | 0          | 0          | 0      | 0      |           0 |   0 |      0 |             0 |
+// | 27  | 28  | 0      | 0            | 27           | x1         | x2         | 0    | 28   | y1         | y2         | 0      | 1      |           4 |   1 |      1 |             1 | <== lookup here
```

Comment:
Hehe, I like the 'lookup here'. But it would be good to specify that `a,b,result,sel_gt` should be used in the lookup? (is that right?)